src/ess.jl

```diff
@@ -311,43 +342,18 @@ function ess_rhat(
     return ess, rhat
 end
 
-"""
-	copyto_split!(out::AbstractMatrix, x::AbstractMatrix)
-
-Copy the elements of matrix `x` to matrix `out`, in which each column of `x` is split.
+function ess_rhat_bulk(x::AbstractArray{<:Union{Missing,Real},3}; kwargs...)
```

Comment:
Since in the next breaking release, my goal is to split `ess_rhat` into `ess` and `rhat`, perhaps it's better to just declare `ess_bulk` and `rhat_bulk` here.